controllers/FFController.java

```diff
@@ -0,0 +1,79 @@
+package org.frc5587.lib.controllers;
+
+public class FFController {
+    public final double kS;
+    public final double kCos;
+    public final double kG;
+    public final double kV;
+    public final double kA;
+
+    /**
+    * sets all given values for:
+    * @param kS static gain
+    * @param kCos gravity
+    * @param kG gravity
+    * @param kV velocity gain
+    * @param kA acceleration gain
+    */
+    public FFController(double kS, double kCos, double kG, double kV, double kA) {
+        this.kS = kS;
+        this.kCos = kCos;
+        this.kG = kG;
+        this.kV = kV;
+        this.kA = kA;
+    }
+
+    /**  
+    * use this for an elevator
+    * all values should be in units that will be determined by how you caluclated them in characterization.
+    * @param velocity the subsystem's velocity in units per second 
+    * (account for encoder counts per revolution and gearing BEFORE passing this parameter) 
+    * @param acceleration acceleration in units per second squared 
+    */
+    public double calculateElevator(double velocity, double acceleration) {
+        return kS * Math.signum(velocity) + kG + kV * velocity + kA * acceleration;
+    }
+
+    public double calculateElevator(double velocity) {
+        return calculateElevator(velocity, 0);
+    }
+
+    /**  
+    * use this for an elevator, or any FPIDSubsystem that does not require an angle gain
```

Comment:
you have a comment here that says to use this for an elevator, but then proceed to have arm methods?